backend/vaccine_manager/models.py

```diff
@@ -0,0 +1,39 @@
+from sqlalchemy import Column, Integer, String, Date, ForeignKey, UniqueConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+class User(Base):
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    name = Column(String, nullable=False)
+    email = Column(String, nullable=False)
+
+    families = relationship('Family', back_populates='user')
+
+class Family(Base):
+    __tablename__ = 'families' 
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    created = Column(Date)
```

Comment:
should this be a datetime instead? I think it would give us more detail that could be useful.

Reply:
good call, fixed!